docs/source/PCs/ARM/RK3399/OS_Downloads/Resources/CS19108R125_R133_R156_OS_Download

```diff
@@ -1,19 +1,13 @@
-.. _CS19108R125-android:
-
 Android7.1
 ----------
 
 | :download:`prebuilt-rk3399v1-cs19108r-edp-android7-ota-emmc-20220217.img <https://chipsee-tmp.s3.amazonaws.com/mksdcardfiles/RK3399/12.5/Android7.1/prebuilt-rk3399v1-cs19108r-edp-android7-ota-emmc-20220217.img>`
 
-.. _CS19108R125-debian:
-
 Debian10
 --------
 
 | :download:`prebuilt-rk3399v1-cs19108r-edp-debian10-emmc-20220317.img <https://chipsee-tmp.s3.amazonaws.com/mksdcardfiles/RK3399/12.5/Debian10/prebuilt-rk3399v1-cs19108r-edp-debian10-emmc-20220317.img>`
 
-.. _CS19108R125-linuxQt:
-
```

Comment:
Please do not remove the unique labels for any of the OS downloads since those are used in the software documentation pages.

Reply:
You are right! Although we don't have software doc for RK3399 ATM, we will in the future.
I have rewritten this part, check if you like the changes!

Reply:
Nice!